### PR TITLE
Create sura_names.xml

### PR DESCRIPTION
**Pull request by habibr**
Preparing for indonesian surah names translation.

### CODE REVIEW

File: app/src/main/res/values-in/sura_names.xml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    <item>An-Nashr</item>
+    <item>Al-Lahab</item>
+    <item>Al-Ikhlas</item>
+    <item>Al-Falaq</item>
+    <item>An-Nas</item>
+  </string-array>
+
+  <!-- If sura name has no translation, please add empty double quotes to the
+       respective item body to keep Android Studio happy -->
+  <string-array name="sura_names_translation">
```
Comment (ozbek) [inline app/src/main/res/values-in/sura_names.xml]: You do not seem to be translating sura names into Indonesian at all. Please remove this array.

Comment (habibr) [inline app/src/main/res/values-in/sura_names.xml]: No, I have not. Sorry, it was too quick to press the pull request button.

**Comment by ahmedre** [comment]
`sura_names_translation` looks good, but can you remove `sura_names` since they're the exact same as the ones in `values` so no need to duplicate?

**Comment by ozbek** [comment]
`sura_names` looked the same to me also when I first saw them, but not all of them are the same :)

**Comment by ahmedre** [comment]
ah good catch, jazakAllah khairan merging this

**Comment by habibr** [comment]
Yes, the english and indonesian surah names are similar but different.

**Pull request merged**